turbinia/workers/workers_test.py

```diff
@@ -213,7 +219,7 @@ def testTurbiniaTaskEvidenceValidationFailure(self, evidence_decode_mock):
     test_result = self.task.run_wrapper(test_evidence.__dict__)
     test_result = TurbiniaTaskResult.deserialize(test_result)
     self.assertFalse(test_result.successful)
-    self.assertIn('validation failed', test_result.status)
+    self.assertIn('TestStatus', test_result.status)
```

Comment:
What changed here?  The validation failure message should still be part of the status, right?  Otherwise I think this is a legit breakage of the test here (though I'm not really sure why your changes would affect this unless it never gets to this point for some reason).

Reply:
Reverted the changes. Test fails with the following error:

```
FAIL: Tests Task fails when evidence validation fails.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/google/home/amisi/Documents/env/lib/python3.7/site-packages/mock/mock.py", line 1330, in patched
    return func(*args, **keywargs)
  File "/usr/local/google/home/amisi/Documents/turbinia/turbinia/workers/workers_test.py", line 222, in testTurbiniaTaskEvidenceValidationFailure
    self.assertIn('validation failed', test_result.status)
AssertionError: 'validation failed' not found in 'TurbiniaTask Task failed with exception: [stat: path should be string, bytes, os.PathLike or integer, not NoneType]'
```

Reply:
Yeah, this looks like it's because something else caused the task to fail before the validation failed it in the expected way for this test.  I suspect that adding the preprocessing mock fixed this, but not 100% sure.